### ISSUE TITLE
Update the blend files to the latest blender version

### ISSUE THREAD

**Issue by drcd1** [drcd1/GodotSimpleExplosionVFX#5]

**Comment by fire** [comment]
I couldn't get a result anywhere close to the example. Maybe I did something wrong on the latest blender 2.91.2

**Comment by drcd1** [comment]
Currently the files only work with Blender 2.80 (and possibly 2.81). Download here: https://download.blender.org/release/Blender2.80/
The blender fluid simulator was overhauled in 2.82, and I haven't learned how to use it yet

**Comment by fire** [comment]
As you're aware of, Godot Engine doesn't support Blender < 2.83.

**Comment by drcd1** [comment]
> 
> 
> As you're aware of, Godot Engine doesn't support Blender < 2.83.

Yes, but I only use Blender to render the sprites as pngs, so Godot doesn't have to support it directly (unlike models and animations). Overwrites to the .png files are instantly reflected in Godot (after a re-import). I just provided the .blend files so that the users can create their own explosion sprites if they so wish, otherwise they're free to just use the rendered sprites I provide, which can be mixed and changed in a variety of ways.

Nevertheless, it is always nice to have things updated to run on the latest version.

**Comment by fire** [comment]
I appreciate the sample though. Do you know why it's wrong?
